samples/src/main/java/com/squareup/sample/launcher/SampleLauncherApp.kt

```diff
@@ -56,7 +62,41 @@ import com.squareup.sample.R
   SampleLauncherApp()
 }
 
-private fun Context.launchSample(sample: Sample) {
-  val intent = Intent(this, sample.activityClass.java)
-  startActivity(intent)
+@Composable private fun SampleItem(sample: Sample) {
+  // See https://issuetracker.google.com/issues/156875705.
+  @Suppress("DEPRECATION")
+  val rootView = (OwnerAmbient.current as AndroidOwner).view
+
+  /**
+   * [androidx.ui.core.LayoutCoordinates.globalBounds] corresponds to the coordinates in the root
+   * Android view hosting the composition.
+   */
+  val globalBounds = remember { Ref<PxBounds>() }
+
+  ListItem(
+      text = sample.name,
+      secondaryText = sample.description,
+      singleLineSecondaryText = false,
+      modifier = Modifier.onPositioned { globalBounds.value = it.globalBounds },
+      onClick = { launchSample(sample, rootView, globalBounds.value) }
+  )
+}
+
+private fun launchSample(
+  sample: Sample,
+  rootView: View,
+  sourceBounds: PxBounds?
+) {
+  val context = rootView.context
+  val intent = Intent(context, sample.activityClass.java)
+  val options: Bundle? = sourceBounds?.let {
+    makeScaleUpAnimation(
```

Comment:
Now you're just showing off.

Reply:
Couldn't help myself. It's like a real launcher! 😅